src/core/connected_clients.rs

```diff
@@ -12,18 +12,18 @@ pub struct ConnectedClients(Vec<ClientId>);
 
 impl ConnectedClients {
     pub(crate) fn add(&mut self, client_id: ClientId) {
-        debug!("adding connected `{client_id:?}`");
+        debug!("adding connected `{client_id}`");
```

Comment:
This changes from `"adding connected ClientId(10)"` to `"adding connected 10"`, which is not what we want.

Reply:
Sure, I wasn't super confident that change was wanted in the output.